core/controllers/reader.py

```diff
@@ -300,7 +300,127 @@ def get(self, exploration_id):
 
 
 class StorePlaythroughHandler(base.BaseHandler):
-    """Handles a useful playthrough coming in from the frontend to store it."""
+    """Handles a useful playthrough coming in from the frontend to store it. If
+    the playthrough already exists, it is updated in the datastore.
+    """
+
+    def __init__(self, request, response):
+        """Method that initializes member variables for the handler."""
+        super(StorePlaythroughHandler, self).__init__(request, response)
+        self.exp_issues = None
+        self.issue_schema_version = None
+
+    def _is_issue_in_exp_issues_model(
+            self, exp_issues, issue_type, customization_args):
+        """Checks whether the given issue exists in the unresolved issues list
+        of the exploration issues model.
+
+        Args:
+            exp_issues: ExplorationIssues. The exploration issues domain object.
```

Comment:
The only value you provide for this is `self.exp_issues`, which is available in this method. Just use it directly instead.

Reply:
Done.